src/app/utils/string-validators.ts

```diff
@@ -0,0 +1,63 @@
+export function containNameCharactersOnly(str: string): boolean {
+	const testRegExp: RegExp = /^([a-zA-Z0-9_]+)$/;
+	const match: RegExpExecArray = testRegExp.exec(str);
+	return (match) ? true : false;
+}
+
+export function containAlphabetsOnly(str: string): boolean {
+	const testRegExp: RegExp = /^([a-zA-Z]+)$/;
+	const match: RegExpExecArray = testRegExp.exec(str);
+	return (match) ? true : false;
+}
+
+export function isValidApiAcceptedDateString(str: string): boolean {
```

Comment:
I'm sure that there would be a library that is capable of this validation. We shouldn't do this from scratch as there are many factors involved.